<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Xiaoping’s CV</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Xiaoping Wu</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="about.html">
    <span class=" about"></span>
     
    About
  </a>
</li>
<li>
  <a href="publication.html">
    <span class=" publications"></span>
     
    Publication
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="mailto:xpwu594419341@gmail.com">
    <span class="fa fa-envelope-o"></span>
     
    Contact me
  </a>
</li>
<li>
  <a href="https://www.linkedin.com/in/xiaoping-wu-5bb316b0/">
    <span class="fa fa-linkedin fa-2x"></span>
     
    linkedin
  </a>
</li>
<li>
  <a href="https://github.com/xiaopingwu-SSI-2021">
    <span class="fa fa-github"></span>
     
    github
  </a>
</li>
<li>
  <a href="https://twitter.com/wu50340717">
    <span class="fa fa-twitter fa-2x"></span>
     
    twitter
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore"><strong>Xiaoping’s CV</strong></h1>

</div>


<p><img src="2.jpeg" style="width:50.0%" /></p>
<div id="profile" class="section level1">
<h1><strong>PROFILE</strong></h1>
<p>Postdoc in <strong>bioinformatics</strong> currently. I am now mainly contributing to research on</p>
<ul>
<li>Genome wide association analysis focus on pregnancy-related outcomes (birthweight and gestational duration) and selected childhood diseases<br />
</li>
<li>Polygenic risk score estimation</li>
<li>Logistic regression analysis on Neuropsychiatric disorders</li>
</ul>
<p>PhD and 5 years postdoc experience in <strong>Quantitative Genetics and Genomics</strong>. I was mainly contributing to research focusing on</p>
<ul>
<li>Quantitative genetics and genomics<br />
</li>
<li>Population genetics<br />
</li>
<li>Statistical genetics</li>
</ul>
<p>I can offer</p>
<ul>
<li>Years of experience on bioinformatics and post-GWAS analysis<br />
</li>
<li>Years of experience research within the field of genome wide association analysis and genomic prediction<br />
</li>
<li>Knowledge about different statistical genetics models - linear mixed model, Bayesian model and logistic regression model, and experience of using different softwares, including R for these models<br />
</li>
<li>Experience in handling large data set, such as whole genome sequence data and different density of SNP-chip data<br />
</li>
<li>Rich experience with R language under Linux environment. I am also familiar with Python<br />
</li>
<li>Rich experience with Snakemake for reproducible research</li>
</ul>
<p>I thrive in diverse, team-based environments, and comfortable with leading and being led.</p>
</div>
<div id="main-research-content" class="section level1">
<h1><strong>MAIN RESEARCH CONTENT</strong></h1>
<ul>
<li>By using a modified linear mixed-model approach, a genome wide association analysis for feet and legs<br />
disorders, young stock survival with imputed whole-genome sequence variants have been performed in cattle<br />
</li>
<li>By checking common haplotypes that never occur in the homozygous state in live animals, some recessive lethal haplotypes have been detected for responsible for early embryonic lethality detected in cattle.<br />
</li>
<li>Investigated the efficiency of two-trait Bayesian variable selection model and two-trait genomic best linear unbiased prediction model combines with genome-wide association analysis information in genomic prediction.<br />
</li>
<li>By using genomic best linear unbiased prediction model and a Bayesian mixture model, the perform of genome wide association analysis were compared for 3 different SNP-chip based marker dataset.<br />
</li>
<li>By using genomic best linear unbiased prediction model and a Bayesian mixture model, I investigated reliability of genomic prediction in various scenarios with regard to relationship between test and training animals and between animals within the training data set.<br />
</li>
<li>By using a least absolute shrinkage and selection operator, a genome wide association analysis has been conducted for body conformation traits in cattle.</li>
</ul>
</div>
<div id="worjing-and-education-experience" class="section level1">
<h1><strong>WORJING AND EDUCATION EXPERIENCE</strong></h1>
<ul>
<li>2020- Vice director, Shijiazhuang, ZhongNongTongChuang (ZNTC) Bio-Technology Co., LTD, Hebei Province, China<br />
</li>
<li>2014 – 2019 Postdoc, Center for Quantitative Genetics and Genomics, Aarhus University, Denmark<br />
</li>
<li>2012 – 2014 Ph.D, Center for Quantitative Genetics and Genomics, Aarhus University, Denmark<br />
</li>
<li>2009 – 2014 successive master-doctor program, Quantitative genetics, China agriculture University, China</li>
</ul>
</div>
<div id="research-funding" class="section level1">
<h1><strong>RESEARCH FUNDING</strong></h1>
<ul>
<li>Joined the GUDP 2016 funding application for “Reducing calf mortality by identifying lethal genes and improving survival index - LiveCalf “.<br />
</li>
<li>Award a scholarship under the State Scholarship Fund in China to pursue my study in Denmark as a joint PhD student from September, 2012 to January, 2014.</li>
</ul>
</div>
<div id="teaching-and-supervision" class="section level1">
<h1><strong>TEACHING AND SUPERVISION</strong></h1>
<ul>
<li>Participated course: Teacher Trainning Programm in English from Module 1 – 4. Aarhus University, August 2016 – January 2017.<br />
</li>
<li>As Teaching assistant for master and PhD course: QTL mapping. Aarhus University. 2016<br />
</li>
<li>As Teaching assistant for bachelor course: Animal breeding. China Agriculture University. 2011.</li>
</ul>
</div>
<div id="language" class="section level1">
<h1><strong>LANGUAGE</strong></h1>
<ul>
<li><strong>Chinese</strong> native<br />
</li>
<li><strong>English</strong> flent<br />
</li>
<li><strong>Danish</strong> learning</li>
</ul>
</div>
<div id="it-skills" class="section level1">
<h1><strong>IT skills</strong></h1>
<ul>
<li><strong>R programming</strong> rich experience<br />
</li>
<li><strong>Linux</strong> rich experience<br />
</li>
<li><strong>snakemake</strong> experienced<br />
</li>
<li><strong>python</strong> familiar</li>
</ul>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
